sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -107,6 +107,8 @@ class ServiceBusReceiver(
      specified Topic the client connects to.
     :keyword Optional[float] max_wait_time: The timeout in seconds between received messages after which the
      receiver will automatically stop receiving. The default value is None, meaning no timeout.
+     If there is session enabled queue or topic and NEXT_AVAILABLE_SESSION is specified,
+     max_wait_time will wait for that time to receive a message from any session before the operations times out.
```

Comment:
Is this the right place to document this? Feels like the more visible place would be at `ServiceBusClient.get_queue_receiver`?

Reply:
It also applies to topics (subscription_receiver) as well